.vsts-ci.yml

```diff
@@ -82,8 +82,8 @@ stages:
     - template: eng/build.yml
       parameters:
         agentOs: Linux
-        jobName: Build_Fedora_36_Debug_x64
-        container: 'mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-36'
+        jobName: Build_Fedora_39_Debug_x64
+        container: 'mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-39'
```

Comment:
This change seems potentially outside the scope of this work as it is not involved with source build. This is the CI leg for just installer stuff, not specifically source build. I say that because I notice this build leg is failing.

Reply:
That makes sense. I've reversed those changes opened and will open an issue in Installer